crates/client/l1-messages/src/error.rs

```diff
@@ -0,0 +1,37 @@
+use mp_felt::Felt252WrapperError;
+
+#[derive(thiserror::Error, Debug, Eq, PartialEq)]
+pub enum L1MessagesConfigError {
+    #[error("File with L1 Messages Worker config not found: `{0}`")]
+    FileNotFound(String),
```

Comment:
Keep the error in the variant. It's an std::fs error I think. And relly on display for the conversion to string